document/FileFormat.mdk

```diff
@@ -2561,23 +2561,28 @@ If this `annotatedCodeLocation` does not occur within a `codeFlow`, the `importa
 
 If present, the `importance` property shall have one of the following values, with the specified meanings:
 
-* `"essential"`: this location is essential for understanding the code flow.
+* `"important"`: this location is important for understanding the code flow.
 
-* `"normal"`: this location is helpful in understanding the code flow.
+* `"essential"`: this location is essential for understanding the code flow.
 
-* `"nonessential"`: this location contributes to a more detailed understanding of the code flow, but is not normally needed. 
+* `"unimportant"`: this location contributes to a more detailed understanding of the code flow, but is not normally needed. 
 
-If this property is absent, it shall be considered to have the value `"normal"`.
+If this property is absent, it shall be considered to have the value `"important"`.
 
 ~ IsoNote
 A viewer might use this property to offer the user three options for viewing a lengthy code flow:
 
-* A "normal view," which omits locations whose `importance` property is `"nonessential"`.
+* A "normal view," which omits locations whose `importance` property is `"unimportant"`.
 * An "abbreviated view," which displays only those locations whose `importance` property is
 `"essential"`.
 * A "verbose view," which displays all the locations in the code flow.
 ~
 
+### `snippet` property { #annotatedCodeLocation-snippet }
+
+An `annotatedCodeLocation` object may contain a property named `snippet` whose value is a string containing
+the text of the source code lines specified by `annotatedCodeLocation.physicalLocation.region`.
+
```

Comment:
Note that I'm proposing a much more restrictive definition for `annotatedCodeLocation.snippet` than for `result.snippet`. For `result.snippet`, I wrote (following the text you put in the schema), that it could be any source text that "illustrates the result". For `annotatedCodeLocation.snippet`, I'm insisting it be the source code lines specified by `annotatedCodeLocation.physicalLocation.region`. Thoughts?